_posts/2021-03-03-cockpit-239.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 239
+author: lis
+date: 2021-03-03
+tags: cockpit linux
+slug: cockpit-239
+category: release
+summary: Cockpit with restyled updates page
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 239.
+
+### Machines: Allow editing disk cache mode
+
+It's now possible to edit the cache mode for existing virtual machine
+images.  This was previously only possible when creating new images.  In
+order to edit the cache mode, any machines using the volume need to be
+powered off.
```

Comment:
I would just do s/image/disk .
What is done with this functionality is related to attaching disks. `Creating new image` is a different action (the one done thru `qemu-img`). 
The difference is that when you are attaching and editing disk, you already have an existing image. Then you only attach/add this image to a VM as a disk (and there you can configure such things as a cache mode)